docs/04-api-reference/front-commerce-remix/graphql-over-http.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: "GraphQL over HTTP"
+sidebar_position: 2
+description:
+  "Your application's unified GraphQL schema can be exposed over HTTP in
+  production or for development purposes only. This guide explains the different
+  options and patterns available to expose a GraphQL HTTP endpoint."
+---
+
+<p>{frontMatter.description}</p>
+
+## The `/graphql` route
+
+By default, Front-Commerce exposes a GraphQL HTTP endpoint on the `/graphql`
+URL. This endpoint is available in production and development environments.
+
+It uses [GraphQL Yoga](https://the-guild.dev/graphql/yoga-server) under the
+hood, and is configured to use the same unified GraphQL schema as
+[the one used across the application (via `app.graphql`)](/docs/3.x/api-reference/front-commerce-remix/front-commerce-app#appgraphql).
+
+You can use this endpoint to query your application's GraphQL API from your
+frontend or from any other application.
+
+## GraphQL Playground
+
+In development mode, Front-Commerce exposes a
+[GraphiQL](https://github.com/graphql/graphiql) Playground on the
+[`/graphql`](http://localhost:4000/graphql) route. This UI allows you to explore
+your application's GraphQL API using a Documentation Explorer, and to test
+queries and mutations without any additional tool.
+
+<Figure caption="The GraphiQL Playground UI allowing to build queries and execute them when Front-Commerce is in development mode.">
+
+[![Screenshot of the GraphiQL Playground UI](./assets/graphiql-screenshot.png)](./assets/graphiql-screenshot.png)
+
+</Figure>
+
+## Production mode
+
+Several GraphQL DX features are disabled in production mode, in order to improve
+performance and security:
+
+- GraphiQL Playground
+- GraphQL introspection
+- GraphQL fields suggestions
+
+:::info
+
+By production mode, we mean that the `FRONT_COMMERCE_ENV` environment variable
+is set to `production`. This is what we recommend for production sites (but not
+needed for staging or local environments).
+
+:::
+
+## How-to customize the HTTP GraphQL endpoint
+
+You may want to customize how the GraphQL endpoint is exposed over HTTP. For
+example, you may want to expose it for logged in users only, to disable it or to
+expose it on a different route. This section explains some of the options
+available to you.
+
+### Expose the GraphQL endpoint on a different route
+
+The default `/graphql` route is registered by the `@front-commerce/remix`
+package. You can reuse its loader/action in another route to expose the GraphQL
+endpoint on a different URL.
+
+You will then import the default loader and action from the
+`@front-commerce/remix/playground.server` package export. A minimal route
+exposing the GraphQL endpoint on the `/my-other-endpoint` would look like this:
+
+```ts title="/app/routes/my-other-endpoint.ts"
+export { loader, action } from "@front-commerce/remix/playground.server";
+```
```

Comment:
thought: Should we precise that when doing so, the `/graphql` endpoint would still be available (i.e. it doesn't "move" the graphql endpoint, it just "copies" it)?